### ISSUE TITLE
macos Copy command not working

### ISSUE THREAD

**Issue by annard** [warpdotdev/Warp#4803]
### Describe the bug

When I select text in a terminal tab and try to copy it nothing gets put on the clipboard. The old content of the clipboard is not affected. This happens in version v0.2024.04.16.08.02.stable_00 and did not happen in the previous version.
This also happens regardless of the preference setting for copy on select and restarting the app doesn't make a difference.
It happens on an M2 MacBook Pro with Sonoma 14.4.1 (23E224).

### To reproduce

1. Select some text in an active terminal
2. Copy (either through Command-C or the menu Edit>Copy)
3. Paste in the terminal (or another app)
4. The selected text is not pasted.

### Expected behavior

I expect the copied text to be pasted.

### Operating system

MacOS

### Operating system and version

14.4.1 (23E224)

### Shell Version

zsh 5.9 (x86_64-apple-darwin23.0)

### Current Warp version

v0.2024.04.16.08.02.stable_00

### Regression

Yes, this bug started recently or with an X Warp version

### Recent working Warp date

April 18, 2024

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by dannyneira** [comment]
Hi @annard sorry to hear about this blocker. If you can please post a video of the issue to better help us understand the bug. I've been unable to reproduce this on Warp on Mac. Are you trying to copy from an alt-screen app or just the normal terminal blocks?

**Comment by annard** [comment]
Hello Danny,

It has been fixed again. The problem was that Command-Copy was redefined. I don't know exactly how this happened, but I do remember that I was playing with the definitions of the different Copy command shortcuts. And somehow the Command-Copy got its shortcut changed to Control-Command-C. I do remember I was trying to use the Edit > Copy menu command and it didn't work either. But now it is working again after I reverted those changes. I apologise for the inconvenience.

Best regards,
Annard
